teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization)
```

Comment:
- [ ] Kindly make sure to also add age, name and parent permission to the initialize constructor. Like:
`def initialize(age, specialization, name = "Unknown", parent_permission = true)`